src/e2e/specs/dashboard.spec.ts

```diff
@@ -302,3 +303,93 @@ test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard  - Payment`, ()
     );
   });
 });
+
+test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard - Footer`, () => {
+  test.beforeEach(async ({ dashboardPage, page }) => {
+    await dashboardPage.open();
+    try {
+      await checkAuthState(page);
+    } catch {
+      console.log("[E2E_LOG] - No fxa auth required, proceeding...");
+    }
+  });
+
+  test("Verify that the site footer is displayed correctly", async ({
+    dashboardPage,
+    page,
+    context,
+  }) => {
+    // link to testrail
+    test.info().annotations.push({
+      type: "testrail",
+      description:
+        "https://testrail.stage.mozaws.net/index.php?/cases/view/2463570",
+    });
+
+    const clickOnATagCheckDomain = async (
+      aTag: Locator,
+      expectedStrings: string[],
+    ): Promise<boolean> => {
+      const href = await aTag.getAttribute("href");
+      if (href === null) return false;
+
+      const target = await aTag.getAttribute("target");
+
+      if (target === "_blank") {
+        //opens link in a new tab
+        const newPage = await context.newPage();
+        await newPage.goto(href);
+        const currentUrl = newPage.url();
+        await newPage.close();
+        return expectedStrings.every((expectedString) =>
+          currentUrl.includes(expectedString),
+        );
+      } else {
+        //opens link in the same tab
+        await page.goto(href);
+        const currentUrl = page.url();
+        await page.goBack();
+        return expectedStrings.every((expectedString) =>
+          currentUrl.includes(expectedString),
+        );
+      }
+    };
+    await dashboardPage.goToDashboard();
+    expect(page.locator("footer a >> img")).toBeTruthy();
+    expect(
+      await clickOnATagCheckDomain(dashboardPage.mozillaLogoFooter, [
+        "mozilla.org",
+      ]),
+    ).toBeTruthy();
+    expect(
+      await clickOnATagCheckDomain(dashboardPage.allBreachesFooter, [
+        "/breaches",
+      ]),
+    ).toBeTruthy();
+    expect(
+      await clickOnATagCheckDomain(dashboardPage.faqsFooter, [
+        "support.mozilla.org",
+        "/kb/",
+        "firefox-monitor-faq",
+      ]),
```

Comment:
The reason why I divide the string up is because the link that is mentioned in testrail opens the same page as the link that is used on stage (which seems to be redirecting to that page, but thus has a different URL). I think we can explicitly check the domain first, and then the path. I am going to add that!

Reply:
Although, do we want to enforce the host? Would the host only be checked for stage - because then it wouldn't just be mozilla.org or its subdomain.

Reply:
We could also try https://playwright.dev/docs/api/class-pageassertions#page-assertions-to-have-url which supports regex so we can ignore host names or subpaths (like localized bits).